library/src/test/java/com/chuckerteam/chucker/internal/support/OkioUtilsTest.kt

```diff
@@ -0,0 +1,56 @@
+package com.chuckerteam.chucker.internal.support
+
+import com.google.common.truth.Truth.assertThat
+import okio.Buffer
+import okio.ByteString.Companion.encodeUtf8
+import org.junit.jupiter.api.Test
+import java.nio.charset.Charset
+
+internal class OkioUtilsTest {
+    @Test
+    fun isPlaintext_withEmptyBuffer_returnsTrue() {
```

Comment:
```suggestion
    fun isProbablyPlainText_withEmptyBuffer_returnsTrue() {
```

This applies to all the other tests